app/models/iam_user.rb

```diff
@@ -73,5 +73,7 @@ def vet360_contact_info
     super
   rescue Faraday::ResourceNotFound
     raise Common::Exceptions::RecordNotFound.new(id: vet360_id)
+  rescue
```

Comment:
Figuring out the error is made harder by Sentry's accessibility issues but could this just rescue [Common::Client::Errors::ClientError](http://sentry.vfs.va.gov/organizations/vsp/issues/42097/events/9ec24ec5f8364c6993636601c15b13b0/?environment=staging&project=3&query=is%3Aunresolved) errors? I'm sure most errors here are that one anyway but there could be a vets-api standard error or system error that would get swallowed (i.e. not reported to Sentry) by the catch all.